tests/operators-ore-order.sql

```diff
@@ -0,0 +1,84 @@
+
+    ore_json_10 := '{"o": ["121212121212597be28282d03415e7714fccd69eb7eb476c70743e485e20331f59cbc1c848dcdeda716f351eb20588c406a7df5fb8917ebf816739aa1414ac3b8498e493bf0badea5c9fdb3cc34da8b152b995957591880c523beb1d3f12487c38d18f62dd26209a727674e5a5fe3a3e3037860839afd801e8dfc77a97f6f7675d1dfba267694bb3ce861127bf8e634f9a08c7ba557a690b5cd6eea691781dc28c3f564232899309322406e48efe76bde50090a6007b7e64ec1a1501b83e850b8c28413b9d410398ea4a0b0682da9fc995002204dc0028cfed3231902972ca934fc39f55b18c6881663ae0313314d8eb7d7490b14955df31f4e2f0cb90d3aeb4d9df3981da50f9dcc901b6d98ef5513cfae6b32bebf79dea1e607a803aee733e8ef19dc392f6241cab423f93bf734034bec55086a9331002998960e83ae2dcca057ee377630028476b2dc11af49d530159eb52f9eed8cc86275dadc948669ce8c357cfd4e3eecd8e364e2dc8abfd2207b00e5bc7ae9759d29f36f51b1e13a1979fc967a1ba2848fd4b58736f1f7d41db82486685c4736b23"]}';
+    ore_json_20 := '{"o": ["1212121212125987e28282d03415e7714fccd69eb7eb476c70743e485e20331f59cbc1c848dcdeda716f351eb20588c406a7df5fb8917ebf816739aa1414ac3b8498e493bf0badea5c9fdb3cc34da8b152b995957591880c523beb1d3f12487c38d18f62dd26209a727674e5a5fe3a3e3037860839afd801b7dfa2e572a8ca3d4e283e60ea4ac66b6cce24bd1dff3ed702ec4b9b6c3e80bf65c58b7e48ef83f6a13241bc37b2f82437d851f1652dc8b4cbfc7da0c8e121908233fd953c09c9cb6685f74924fd50f2cb9f09990132102950c42b31632e1dab147d9cce096ab25ac8b165839d419b2c4f096321bb2cc110688e48f4826392dbb18849a7ccc757eb22d1412f6423e9c5fe2e9a548c880683dc3cb895a160009dcd48af01080f5ad2d001fed5286cd944a0bfef8eed8f25d3db11e954ba5642efc4b1fee5a49cc07fdb342694e2614cb4106422804c0861fa161c7589f2d38cb1708b1fa179a7a5a46f56bf1c3bafd42b1f509362562d29019068f31b55d969daa655ceaa292b0a343cb4aec7f65b71d031c56b9c3ee17832553e8076600228c9"]}';
+    ore_json_30 := '{"o": ["12121212121259f2e28282d03415e7714fccd69eb7eb476c70743e485e20331f59cbc1c848dcdeda716f351eb20588c406a7df5fb8917ebf816739aa1414ac3b8498e493bf0badea5c9fdb3cc34da8b152b995957591880c523beb1d3f12487c38d18f62dd26209a727674e5a5fe3a3e3037860839afd8011d6dae3b7cd932a5e966f91cd1e43f50ff0f444b642edc6ce7d25ac2780f5273df703499a9403f90da8027522af65d0eab400f8584b07c4a203668e6f761a4b2762c1bd4817660b42f03ec13e149851ab46c651f3491f7b744e24bac686048eb755d61ed4644cc77b74b441b11ed825b08bca32b6769469cd37100dfe6aed9eaf315236c00dbe0d3ed0b54162106ec5f35dd7d21a0e081885f378be66243a595d0b162908a39dee2f787a15fa77b68b98dced5ef0f232841d11263833a6fe1a584f027fb46b609ada90ead3c52cf8dd60221df83373314bd1a0208a5fbebc45a02f45caa35e0e6828b331c686c2d0c1ffb39963bc48ea5580c1dab383b896a1c439de753964e17093ca0779e53ea4bac6c05c87ac3539feb7acce624d43c22ac"]}';
+
+
+ -- ------------------------------------------------------------------------------------------------
+
+    -- DESC
+    -- NULLS FIRST is the default for DESC order
+    -- https://www.postgresql.org/docs/current/queries-order.html
+    SELECT id FROM encrypted ORDER BY encrypted_int2 DESC LIMIT 1
```

Comment:
How does `ORDER BY` work here without using the EQL ORE function?

Reply:
I think these might be doing the default sort on `JSON` and the source ciphertexts happen to line up to make it appear like these tests are working.

Reply:
Uh, doh. Yes.